paparazzi/paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityOverlayDetailsView.kt

```diff
@@ -0,0 +1,100 @@
+package app.cash.paparazzi.accessibility
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.graphics.Rect
+import android.graphics.RectF
+import android.text.StaticLayout
+import android.text.TextPaint
+import android.text.TextUtils
+import android.util.TypedValue
+import android.widget.FrameLayout
+import app.cash.paparazzi.accessibility.RenderSettings.DEFAULT_TEXT_COLOR
+import app.cash.paparazzi.accessibility.RenderSettings.toColorInt
+import java.lang.Float.max
+
+class AccessibilityOverlayDetailsView(context: Context) : FrameLayout(context) {
```

Comment:
this view is used to render accessibility details of elements in the captured view passed to paparazzi.
We had to render text and badges via canvas because updates to the accessibility state happen after layout phase and we can't re-trigger layout with our current use of layoutlib. 
Manipulating state in `onPreDraw` was the simplest solution.